src/rhsmlib/dbus/facts/com.redhat.RHSM1.Facts.service

```diff
@@ -0,0 +1,5 @@
+[D-BUS Service]
```

Comment:
Do these need to be in `src/rhsmlib/dbus` ? I would have expected them in `etc-conf`...